### PR TITLE
docs: add popover API blog post

### PR DESCRIPTION
**Pull request by vladitasev**
Read here: https://github.com/SAP/ui5-webcomponents/blob/popover-api-blog/packages/website/blog/releases/popover-api-in-v2.mdx

### CODE REVIEW

**Review by ilhan007 — APPROVED** [review]

File: packages/website/blog/releases/popover-api-in-v2.mdx [hunk 1]
```diff
@@ -0,0 +1,178 @@
+---
+title: UI5 Web Components 2.0 will leverage the popover API
+tags: [release]
+slug: /releases/popover-api-in-v2
+---
+
+UI5 Web Components 2.0 will provide greatly improved popups by taking advantage of the [native browser popover API](https://developer.mozilla.org/en-US/docs/Web/API/Popover_API)
```
Comment (petyabegovska) [inline packages/website/blog/releases/popover-api-in-v2.mdx:7]: Add a period at the end of the sentence.

File: packages/website/blog/releases/popover-api-in-v2.mdx [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+    - Tests no longer need to know how to find the static area item, associated with a given component - everything is directly in the shadow root!
+    - Writing tests is much simplified
+
+#### Cross-framework popup compatibility for the future
+    - Frameworks who use the native browser popover API no longer need to synchronize themselves (negotiate `z-index` values, etc.)
+    - The last popup to be opened will always be on top (guarnateed by the browser)!
+
+### Is it safe to start using the popover API productively?
+    - The current versions of all supported browsers (Chrome, Safari, Edge, and now also Firefox as of version 125) fully support the popover API.
+    - By the time **v. 2.0** is officially released (we are at **v. 2.0-r.c.2** as of writing this blog post) we expect that each major browser will have already released at lest 3 stable versions since the introduction of the popover API.
```
Comment (petyabegovska) [inline packages/website/blog/releases/popover-api-in-v2.mdx]: typo: at lest --> at least

File: packages/website/blog/releases/popover-api-in-v2.mdx [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+
+### What are the **practical** benefits?
+
+#### Simpler and more robust components
+    - Easier to develop and maintain
+    - Everything belonging to a component is now in one place! This includes code logic, HTML and CSS.
+
+#### Enhanced overstyling capabilities for apps
+    - We can now provide [CSS Shadow Parts](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_shadow_parts) also for the "popup part", not just in the "main part" of the component!
+    - [CSS Custom Properties](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties) set on the comopnent will also have effect for its "popup part"!
```
Comment (petyabegovska) [inline packages/website/blog/releases/popover-api-in-v2.mdx:77]: typo: comopnent --> component

File: packages/website/blog/releases/popover-api-in-v2.mdx [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+
+As you can see, the component used to be physically divided in two parts:
+ - The "main" part (the `ui5-date-picker` tag itself) containing the date selection input
+ - The "popover" part (the `ui5-static-area-item` tag, associated with the said date picker) containing the picker (calendar with years/months/days)
+
+### How do UI5 Web Components popups look like in **version 2.x** ?
+
+There is no longer need for a "static area" since the browser now ensures the correct positioning of popups thanks to the popover API.
+
+Example of `ui5-date-picker`'s DOM structure in **v. 2.0**
```
Comment (petyabegovska) [inline packages/website/blog/releases/popover-api-in-v2.mdx:49]: I would rather suggest v2.0, but both are fine.

File: packages/website/blog/releases/popover-api-in-v2.mdx [hunk 5]
```diff
@@ -0,0 +1,178 @@
+---
+title: UI5 Web Components 2.0 will leverage the popover API
+tags: [release]
```
Comment (petyabegovska) [inline packages/website/blog/releases/popover-api-in-v2.mdx]: Add v2 tag also.

**Review by petyabegovska — COMMENTED** [review]
Perfect! Small typos..

File: packages/website/blog/releases/popover-api-in-v2.mdx (hunk 4 above)
Comment (vladitasev) [reply packages/website/blog/releases/popover-api-in-v2.mdx]: Thanks, all done!

**Pull request merged**